templates/single-product/related.php

```diff
@@ -24,7 +24,7 @@
 
 	<section class="related products">
 
-		<h2><?php esc_html_e( 'Related products', 'woocommerce' ); ?></h2>
+		<h2><?php printf( esc_html__( 'Related products', 'woocommerce' ) ); ?></h2>
```

Comment:
This `printf()` is not correct, since there is no placeholder to apply to that string, only `esc_html_e()` is enough here.

Reply:
@claudiosanches  would `esc_html_e__` do the job here?

Reply:
@thiagoghisi there is no `esc_html_e__` function on WordPress, the correct one is just `esc_html_e()` and nothing else.

Reply:
Please check the docs:

- https://developer.wordpress.org/reference/functions/esc_html__/
- https://developer.wordpress.org/reference/functions/esc_html_e/
-http://php.net/manual/en/function.sprintf.php
- http://php.net/manual/en/function.printf.php